samples/SortBenchmark/DiskHelper.fs

```diff
@@ -41,13 +41,12 @@ open Prajna.Tools
 open Prajna.Tools.FSharp
 open Prajna.Tools.StringTools
 open Prajna.Core
-//open Prajna.Service.CoreServices
 
 [<Serializable>]
 type StorageProfile (profileName : string) =
     //member val diskProfile = [|"c:\\sortbenchmark\\";"c:\\sortbenchmark\\";"c:\\sortbenchmark\\";"c:\\sortbenchmark\\";"d:\\sortbenchmark\\";"e:\\sortbenchmark\\";"f:\\sortbenchmark\\"|] with get
-    member val drives = DriveInfo.GetDrives()
-    member val diskProfile = DriveInfo.GetDrives() |> Array.filter(fun di -> di.DriveType = DriveType.Fixed)  |> Array.filter(fun di -> di.TotalSize > 100000000000L) |> Array.map(fun di -> Path.Combine(di.Name,"sortbenchmark").ToLower()+"\\")
+    //member val drives = DriveInfo.GetDrives()
+    member val diskProfile = null with get,set
```

Comment:
Thanks Bruno for the suggestions. I agree. Since SortBenchmark is an individual sample project. Let's just merge this pull request if all other issues are addressed and I can integrate the comments after it is merged. Actually, it's not easy for me to make any changes right now since it's in Sanjeev's branch.